x/liquidity/spec/01_concepts.md

```diff
@@ -4,88 +4,55 @@
 
-- Combines a traditional order book-based exchange system with a Uniswap-like AMM mechanism. This hybrid system deepens liquidity for the token swap marketplace.
+The ESPM model is intended to provide protection against price volatility, transaction ordering vulnerabilities, and losses due to arbitrage, which are widely known vulnerabilities in AMMs such as Uniswap.
+## Batch Execution
 
-- Executes batch-style swaps that minimize front-running risk and sub-second latency competition, thereby protecting ordinary traders.
+The liquidity module uses batch execution methodology. Making deposit, withdrawal, or swap orders are accumulated in a liquidity pool for a pre-defined period, which can be one or more blocks in length. Orders are then added to the pool and executed at the end of the batch. The size of each batch is changeable by using governance parameter called `UnitBatchSize`.
 
-  - The order book accumulates incoming limit orders into a batch.
-  - The liquidity module matches accumulated limit orders and orders from the liquidity pool at an equivalent swap price at each batch execution height.
-  - All limit orders in a batch are treated equally and executed at the same swap price.
+## Price Discovery
 
-## Features of the Liquidity Module
+Swap prices in liquidity pools are determined by the current pool coin reserves and the requested swap amount. Arbitrageurs buy or sell coins in liquidity pools to gain instant profit that results in real-time price discovery of liquidity pools.
 
-The main features of the liquidity module are:
+## Escrow Process
 
-### Liquidity Pool
+The liquidity module uses module account that acts as an escrow account that holds and releases the coin amount during batch execution.
 
-A liquidity pool is a coin reserve that contains two kinds of coins in a trading pair. A unique pool exists for each token pair.
+## Fees
+### PoolCreationFee
 
-A liquidity provider (entity or person) deposits the coins into the liquidity pool and then shares the accumulated swap fees with respect to their pool share. Pool share is represented as possession of pool coins.
+The liquidity module has `PoolCreationFee` parameter that is paid upon pool creation. This param is defined at genesis and it can be updated via the governance proposal. The purpose of this fee is to prevent from creating useless pools and making limitless transactions. This fee goes to community fund.
```

Comment:
> instead of Latin via, let's instead use "by using" (or "using)

That is good point. I tend to use word "via" and I didn't know it is Latin. 

> who is the actor who is being prevented? I used "users" but if we have a better term, we can use

I guess we can use "users". It can be retail traders or liquidity participants maybe?